include/seqan3/io/sequence_file/input.hpp

```diff
@@ -589,9 +592,12 @@ class sequence_file_input
     }
     //!\}
 
+#pragma GCC diagnostic push // avoid deprecation warning of field::seq_qual
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
     //!\brief The options are public and its members can be set directly.
     sequence_file_input_options<typename traits_type::sequence_legal_alphabet,
                              selected_field_ids::contains(field::seq_qual)> options;
```

Comment:
Here for example the deprecated `field:seq_qual` is being used. It seems to me that it would make more sense to remove the usage and replace it with an alternative (since you are deprecating `field:seq_qual`). But you wrap it to avoid deprecation warnings instead.

And I noticed that the PR description

Reply:
Thx! Yes, this doesn't seem correct. I will look into it

Reply:
Okay :) what you did does match the issue description but I also found it a bit strange